windows/src/buildtools/cef-checkout.sh

```diff
@@ -26,6 +26,8 @@ CEF_VERSION=`cat $CEF_VERSION_MD | tr -d "[:space:]"`
 # Note that 14.0 and earlier versions rely on the `master`
 # branch, so that is the default branch checked out in the
 # build configurations
+git reset --hard
+git clean -fd
```

Comment:
I'm a little leery of this; it might be better to check if we are on the right branch already and avoid this kind of shenanigans.